rabbitmq-ballerina/tests/server/docker-compose.yaml

```diff
@@ -0,0 +1,17 @@
+version: "3"
```

Comment:
Don't we add license header for these ?

Reply:
I'm not sure about this. We haven't added in nats also. Should it be added